packages/docusaurus-theme-translations/locales/tk/theme-common.json

```
@@ -0,0 +1,74 @@
+  "theme.blog.paginator.newerEntries": "Täze ýazgylar",
+  "theme.blog.paginator.olderEntries": "Köne ýazgylar",
+  "theme.blog.post.paginator.navAriaLabel": "Blog ýazgysynyň sahypasyndaky nawigasiýa",
+  "theme.blog.post.paginator.newerPost": "Täze ýazgy",
+  "theme.blog.post.paginator.olderPost": "Köne ýazgy",
+  "theme.blog.post.plurals": "{count} ýazgy|{count} ýazgylar|{count} ýazgylary",
+  "theme.blog.post.readMore": "Dowamyny okamak",
+  "theme.blog.post.readMoreLabel": "{title} hakda doly okamak",
+  "theme.blog.post.readingTime.plurals": "{readingTime} min okamak|{readingTime} min okamak|{readingTime} min okamak",
+  "theme.blog.sidebar.navAriaLabel": "Soňky blog barada nawigasiýa",
+  "theme.blog.tagTitle": "\"{tagName}\" bilen {nPosts} ýazgylar",
+  "theme.colorToggle.ariaLabel": "Gijeki ýa-da gündizki temany saýlamak (häzirki wagtda {mode} ulanylýar)",
+  "theme.colorToggle.ariaLabel.mode.dark": "Gijeki tema",
+  "theme.colorToggle.ariaLabel.mode.light": "Gündizki tema",
+  "theme.common.editThisPage": "Bu sahypany üýtgetmek",
+  "theme.common.headingLinkTitle": "{heading} sahypa göni geçiň",
+  "theme.common.skipToMainContent": "Esasy mazmuna geç",
+  "theme.docs.DocCard.categoryDescription.plurals": "{count} element|{count} elementler|{count} elementler",
+  "theme.docs.breadcrumbs.home": "Baş sahypa",
+  "theme.docs.breadcrumbs.navAriaLabel": "Häzirki sahypa üçin nawigasiýa",
+  "theme.docs.paginator.navAriaLabel": "Dokument sahypasy",
+  "theme.docs.paginator.next": "Indiki sahypa",
+  "theme.docs.paginator.previous": "Öňki sahypa",
+  "theme.docs.sidebar.closeSidebarButtonAriaLabel": "Nawigasiýa panelini ýap",
+  "theme.docs.sidebar.collapseButtonAriaLabel": "Gapdal paneli ýygnamak",
+  "theme.docs.sidebar.collapseButtonTitle": "Gapdal paneli ýygnamak",
+  "theme.docs.sidebar.expandButtonAriaLabel": "Gapdal paneli giňeltmek",
+  "theme.docs.sidebar.expandButtonTitle": "Gapdal paneli giňeltmek",
+  "theme.docs.sidebar.navAriaLabel": "Dokumentleriň gapdal paneli",
+  "theme.docs.sidebar.toggleSidebarButtonAriaLabel": "Navigasiýa panelini üýtgetmek",
+  "theme.docs.tagDocListPageTitle": "\"{tagName}\" bilen {nDocsTagged} dokumentler",
+  "theme.docs.tagDocListPageTitle.nDocsTagged": "1-nji sahypa|{count}-nji(y) sahypa|{count}-nji(y) sahypa",
+  "theme.docs.versionBadge.label": "Wersiýa: {versionLabel}",
+  "theme.docs.versions.latestVersionLinkLabel": "iň soňky wersiýa",
+  "theme.docs.versions.latestVersionSuggestionLabel": "Iň soňky dokumentasiýa {latestVersionLink} ({versionLabel}) sahypasynda.",
+  "theme.docs.versions.unmaintainedVersionLabel": "{versionLabel} wersiýasy üçin {siteTitle} dokumentleri indi goldanmaýar.",
+  "theme.docs.versions.unreleasedVersionLabel": "Bu {siteTitle} {versionLabel} wersiýasy üçin geljekki dokumentler.",
+  "theme.lastUpdated.atDate": " {date}",
+  "theme.lastUpdated.byUser": " {user} tarapyndan",
+  "theme.lastUpdated.lastUpdatedAtBy": "Soňky täzelenme {atDate} {byUser} tarapyndan",
```

The extra spaces are already included in the `{atDate}` and `{byUser}` variables declared 1-2 lines above. Both variables are optional and can be omitted. Here if both variables are present, we end up with duplicate spaces, so you should rather remove the space just before.

```suggestion
  "theme.lastUpdated.lastUpdatedAtBy": "Soňky täzelenme{atDate}{byUser} tarapyndan",
```